demo/scroll-into-view/scroll-into-view.js

```diff
@@ -0,0 +1,7 @@
+document.getElementById('scroll-button').addEventListener('click', () => {
+  document.getElementById('scroll-element').scrollIntoView();
+  });
```

Comment:
nit: whitespace here doesn't look right. 
```suggestion
});
```